public-docs/intro-glance.md

```diff
@@ -0,0 +1,80 @@
+
+## Localization and Internationalization
+
+We have [internationalization support](i18n.md) for dozens of languages, including right-to-left formats! We also support currency conversion and localized currency formats.
+
+## Search Engine Optimization
+
+Since 2014 [Google has indexed JavaScript when crawling websites](https://webmasters.googleblog.com/2014/05/understanding-web-pages-better.html). Reaction, however, offers page pre-rendering and product detail page metatag generation out of the box to ensure products are indexed well for web crawlers and search engines:
+
+1. [Prerender.io](https://prerender.io/): Reaction includes integration with Prerender.io out of the box. Prerender.io is a commercial service that will generate static renderings of the application for search engines. All you have to do is provide a key to your site and Prerender will handle the pre-rendering.
+2. [Meteor Chrome Headless Spiderable](https://github.com/artlimes/meteor-chrome-headless-spiderable) with the [Reaction Commerce Caching Plugin](https://github.com/artlimes/reaction-commerce-caching-plugin): This updated Meteor package uses Google's Headless Chrome to crawl pages. Combined with the Reaction-specific caching plugin, the Spiderable package will allow Reaction pages to be crawled by search engines.
+3. [dom](https://github.com/reactioncommerce/reaction/blob/master/imports/plugins/core/dom/client/dom.js): Automatically adds `<meta>` tags are for products using the [dom](https://github.com/reactioncommerce/reaction/blob/release-1.8.0/imports/plugins/core/dom/client/dom.js) core package since 1.8, which uses the title, description and `details` fields of the product to render SEO-friendly data. You may need to use a tool like [SEO Inspector](https://chrome.google.com/webstore/detail/seo-inspector/iejckekdjogeeilmllnabmgkbbmedeal?hl=en) to see this data. Read more on [SEO and Metadata](seo-metadata.md).
+4. The storefront UI starter kit app uses NextJS framework with server side rendering enabled.
+
+Read more about [SEO and Reaction Commerce](https://blog.reactioncommerce.com/how-our-javascript-platform-handles-seo/) in our latest blog post.
+
+## Performance and Scaling
+
+We've done performance testing that mimics high-volume shops, and we're constantly working to improve performance and scalability both in terms of a production deployment and providing a fast and pleasant experience for developers.
+
+## Payments
+
+Reaction currently ships with one production-ready payment plugin: Stripe. If you use Stripe for payments, you can get up and running quickly. If you need to use a different payment service, you'll have to build a custom payment plugin to interface with their API, but we promise it is a painless procedure.
+
```

Comment:
Do we and is it?

Reply:
It seems painless to me, but maybe we shouldn't promise. 😄

Reply:
I don't it's actually clear now if you wanted to integrate a payment provider into a Meteor/NextJS environment exactly how you do that? Stripe is pretty hard-coded into the Starter kit. The payment provider tutorial is pretty much Meteor focused and server-side processing focused.

Reply:
I'll be updating the payment tutorial in this PR before it's merged. So it will be clear. There are a half dozen more articles I need to rewrite before this gets merged.

Reply:
In 2.1/next starterkit release we'll be adding the example payment UI and then it should be clearer, too.